tests/ci_build/test_python.sh

```diff
@@ -60,6 +60,7 @@ case "$suite" in
   cpu)
     source activate cpu_test
     install_xgboost
+    export RAY_OBJECT_STORE_ALLOW_SLOW_STORAGE=1
```

Comment:
@trivialfis  It will be probably better to increase the size of `/dev/shm`, as recommended by the error message. I can take care of that. WDYT?

Reply:
That's a better solution.  Please help increasing the shared memory size.

Reply:
@hcho3 Maybe we can just go with the current setting and don't think about it, we are not actually doing anything with ray anyway, it's just somehow in the dependency tree.